advocacy_docs/edb-postgres-ai/hybrid-manager/using_hybrid_manager/upgrading/pg_major/upgrade_major.mdx

```diff
@@ -35,6 +35,9 @@ Now check the cluster and database metrics and make sure everything is operating
 
 ## Rollback considerations
 
-There is no automatic rollback available with this approach and performing a downgrade is not recommended. 
-To undo the minor upgrade, restore from the latest backup in a new cluster running the preferred minor version and switching application workloads over to the new cluster.
-You can then delete the PG minor upgraded cluster or troubleshoot it further.
+Rollback after a major version upgrade is not possible with these types of clusters.  To undo the major upgrade, 
```

Comment:
for HCP 1.2
Recommended sequence of steps to be followed.
* Take an on demand backup called B1. 
* Restore a PG from that backup B1 called `R1` as this can be used if Major upgrade fails, make sure the restore is sucessful and the cluster is healthy
* Trigger Major version upgrade. 
* If the upgrade is successful delete the cluster `R1`. 

HCP 1.3
* Take an on-demand backup before triggering Major version upgrade
* Trigger Major version upgrade. 
* if the upgrade fails use the previous backup to restore.